tools/packaging/nfpm.jsonnet

```diff
@@ -0,0 +1,90 @@
+local overrides = {
+  logcli: {
+    description:
+      |||
+        LogCLI is the command-line interface to Loki.
+        It facilitates running LogQL queries against a Loki instance.
+      |||,
+  },
+
+  'loki-canary': {
+    description: 'Loki Canary is a standalone app that audits the log-capturing performance of a Grafana Loki cluster.',
+  },
+
+  loki: {
+    description: |||
+      Loki is a horizontally-scalable, highly-available, multi-tenant log aggregation system inspired by Prometheus. 
+      It is designed to be very cost effective and easy to operate. 
+      It does not index the contents of the logs, but rather a set of labels for each log stream.
+    |||,
+    contents+: [
+      {
+        src: './tools/packaging/loki.service',
+        dst: '/etc/systemd/system/loki.service',
+      },
+      {
+        src: './cmd/loki/loki-local-config.yaml',
```

Comment:
This config file configures Loki to write to `/tmp/loki`. Not sure if this is the ideal default location, but I think it's good enough for a first iteration.